mmv1/products/cloudfunctions2/terraform.yaml

```
@@ -0,0 +1,34 @@
+# Copyright 2019 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Provider::Terraform::Config
+overrides: !ruby/object:Overrides::ResourceOverrides
+  CloudFunction:  !ruby/object:Overrides::Terraform::ResourceOverride
+    id_format: 'projects/{{project}}/locations/{{region}}/functions/{{cloud_function}}'
```

```suggestion
    id_format: 'projects/{{project}}/locations/{{region}}/functions/{{name}}'
```

Same below- this looks for a field on the resource ultimately, and `cloud_function` isn't one. `name` should be correct (we may need to fiddle with some overrides to get the right format, depending on the API response)

---

I'm running into issues with 404s on the test files, not sure I'm quite understanding this piece to ensure the API request/responses are coming through correctly. I updated it from cloud_function to name. I was taking after the Gen1 GCF configuration, but unsure if those test cases are correct.